app/controllers/concerns/deliver_now_with_error_handling.rb

```diff
@@ -0,0 +1,16 @@
+module DeliverNowWithErrorHandling
```

Comment:
Flag/reminder that we might not want to keep this implemented as a controller concern